src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -254,6 +282,41 @@ public async Task<HttpResponseMessage> HandleRequestAsync(HttpRequestMessage req
             return response;
         }
 
+        private async Task<HttpResponseMessage> HandleDeleteHistoryByIdRequestAsync(
+            HttpRequestMessage request,
+            string instanceId)
+        {
+            DurableOrchestrationClientBase client = this.GetClient(request);
+            await client.PurgeInstanceHistoryAsync(instanceId);
+            var response = request.CreateResponse(HttpStatusCode.OK);

```

Comment:
Should we return a different status code (say, `404 (Not Found)`) if the orchestration instance to be purged doesn't exist? This is more in line with REST API conventions.

Reply:
@kashimiz great point.

I will fix it.

Thank you!